README.md

```
@@ -295,8 +188,8 @@ We also scale ideally to >98% of LLNL El Capitan.
 * Ideal weak scaling to 100% of the largest GPU and superchip supercomputers
  	* \>43K AMD APUs (MI300A) on [LLNL El Capitan](https://hpc.llnl.gov/hardware/compute-platforms/el-capitan)
    	* \>3K AMD APUs (MI300A) on [LLNL Tuolumne](https://hpc.llnl.gov/hardware/compute-platforms/tuolumne)
-	* \>33K AMD GPUs (MI250X) on [OLCF Frontier](https://www.olcf.ornl.gov/frontier/) 
-	* \>10K NVIDIA GPUs (V100) on [OLCF Summit](https://www.olcf.ornl.gov/summit/) 
+	* \>33K AMD GPUs (MI250X) on [OLCF Frontier](https://www.olcf.ornl.gov/frontier/)
```

markdownlint-cli2 (0.20.0)

[warning] Inconsistent indentation for list items at the same level
Expected: 2; Actual: 1

(MD005, list-indent)